db/schema/patch-002-003.sql

```diff
@@ -0,0 +1,5 @@
+-- add locale to accounts table
+
+ALTER TABLE accounts ADD COLUMN locale VARCHAR(255);
+
+UPDATE dbMetadata SET value = '3' WHERE name = 'schema-patch-level';
```

Comment:
Hmm...maybe we can't have this in a config setting after all, if we're hard-coding `schema-patch-level` in all our patches..?